crm_phonecall/models/crm_phonecall.py

```diff
@@ -217,3 +217,16 @@ def action_button_convert2opportunity(self):
         opportunity_id = opportunity.create(self._prepare_opportunity_vals())
         self.write({"opportunity_id": opportunity_id.id, "state": "done"})
         return opportunity_id.redirect_lead_opportunity_view()
+
+    def _get_view(self, view_id=None, view_type="form", **options):
+        """This is done for injecting the attribute depending on the user's permissions
+        as moving the group of the view attribute element didn't work and no similar examples
+        of use where found either"""
```

Comment:
```suggestion
        """Inject group check to make the tree editable or not. Apparently it can't be done
        by pure view inheritance and attribute overriding"""